osu.Desktop/OsuGameDesktop.cs

```diff
@@ -0,0 +1,60 @@
+﻿using osu.Game;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using osu.Framework.Platform;
+using osu.Framework.Desktop.Platform;
+using osu.Game.Database;
+
+namespace osu.Desktop
+{
+    class OsuGameDesktop : OsuGame
+    {
+        public OsuGameDesktop(string[] args = null)
+            : base(args)
+        {
+
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+        }
+
+        public override void SetHost(BasicGameHost host)
+        {
+            base.SetHost(host);
+            var desktopWindow = host.Window as DesktopGameWindow;
+            if(desktopWindow != null)

```

Comment:
maybe, but feel that's unfitting in this context because it's a precondition to one specific action, not to the entire logical operation of SetHost() (ie: the existence of a desktop window only affects drag&drop, not other stuff that SetHost may or may not need to do in the future), so I feel it's obscuring intent if I reduce nesting with an early return in this case. Won't do, unless that's a reason to deny the PR.